### ISSUE TITLE
Unique population statistics for each time step?

### ISSUE THREAD

**Issue by naveenariva** [OlavHN/bnlstm#7]
Thanks for the implementation. Does this implementation compute population statistics for each time step? I  may be missing something, but It seems to be me that it is shared across time steps and I don't think this is what the Recurrent Batch Norm paper was doing.

**Comment by yansjbuaa** [comment]
I have a same question. Do you have some thoughts about that? When I run bn-lstm in testing process,  the results is terrible.

**Comment by batzner** [comment]
Any update on this? Would be interesting to know how you solved the problem that the `RNNCell`'s `__call__` method doesn't have access to the current time step in TensorFlow. 

Edit:
For anyone trying to figure out how to compute population statistics for each time step, I can recommend [this gist](https://gist.github.com/spitis/27ab7d2a30bbaf5ef431b4a02194ac60). The solution is to pass the current time step via the `__call__` method's `state` parameter.
